tests/unit/ProcessMaker/Nayra/Bpmn/PathTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace ProcessMaker\Nayra\Bpmn;
+
+use PHPUnit\Framework\TestCase;
+use ProcessMaker\Nayra\Bpmn\Path;
+
+class PathTest extends TestCase
+{
+    //Tests that a path stores and returns correctly its elements
```

Comment:
Use documentation block instead of inline comments, to document the method.
`/**`
`  *`
` */`